consensus/src/marshal/mod.rs

```diff
@@ -59,12 +59,12 @@ pub mod cache;
 pub mod config;
 pub use config::Config;
 pub mod finalizer;
-pub use finalizer::Finalizer;
```

Comment:
This didn't need to be exported (in fact would be confusing if you looked at it).